addons/web/data/report_layout.xml

```diff
@@ -1,36 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <record id="report_layout_standard" model="report.layout">
-            <field name="name">Standard</field>
-            <field name="view_id" ref="web.external_layout_standard"/>
-            <field name="image">/web/static/img/preview_standard.png</field>
-            <field name="pdf">/web/static/pdf/preview_standard.pdf</field>
-        </record>
-        <record id="report_layout_background" model="report.layout">
+
+         <record id="report_layout_background" model="report.layout">
             <field name="name">Background</field>
             <field name="view_id" ref="web.external_layout_background"/>
             <field name="image">/web/static/img/preview_background.png</field>
             <field name="pdf">/web/static/pdf/preview_background.pdf</field>
         </record>
+
+        <record id="report_layout_standard" model="report.layout">
+            <field name="name">Light</field>
+            <field name="view_id" ref="web.external_layout_standard"/>
+            <field name="image">/web/static/img/preview_standard.png</field>
+            <field name="pdf">/web/static/pdf/preview_standard.pdf</field>
+        </record>
+
         <record id="report_layout_boxed" model="report.layout">
             <field name="name">Boxed</field>
             <field name="view_id" ref="web.external_layout_boxed"/>
             <field name="image">/web/static/img/preview_boxed.png</field>
             <field name="pdf">/web/static/pdf/preview_boxed.pdf</field>
         </record>
         <record id="report_layout_clean" model="report.layout">
-            <field name="name">Clean</field>
+            <field name="name">Bold</field>
```

Comment:
are you sure about the changes ?

Reply:
@kebeclibre To be honest I think clean could be better than bold. The reason for the renaming of those is that it was wanted to set background (the default style) as first in the list. And I didn't like the fact that the style "Standart" was no longer the first (because it feels to me that if you call something standart, it should be displayed first as default). So the compromise with the PO was to give new names to avoid this line of thoughts. But if one is not of your liking we can change :)